### PR TITLE
panamax: new package and service

### PR DESCRIPTION
**Pull request by offlinehacker**
This is work by @matejc, but simplifed, refactored & tested by me and ready for merge. This also requires https://github.com/NixOS/nixpkgs/pull/5486 or similar pull request to be merged, for panamax to be usefull.

### CODE REVIEW

File: pkgs/applications/networking/cluster/panamax/ui.nix [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  buildInputs = [ makeWrapper ruby openssl sqlite (rubygemsFun ruby) ];
+
+  setSourceRoot = ''
+    mkdir -p $out/share
+    cp -R git-export $out/share/panamax-ui
+    export sourceRoot="$out/share/panamax-ui"
+  '';
+
+  postPatch = ''
+    find . -type f -iname "*.haml" -exec sed -e 's|CoreOS Journal|NixOS Journal|g' -i "{}" \;
```
Comment (offlinehacker) [inline pkgs/applications/networking/cluster/panamax/ui.nix]: image

**Comment by offlinehacker** [comment]
Failed builds are related to some other depency...

**Comment by lethalman** [comment]
Seems ok to me, any reason why you didn't merge?

**Comment by offlinehacker** [comment]
I'm waiting for #5486 to be merged

**Comment by matejc** [comment]
fixed and rebased to master

**Comment by offlinehacker** [comment]
Cool, i'm merging this.

**Pull request merged**